jenkins/deployment/community/cortx-community-deploy.groovy

```diff
@@ -155,12 +155,14 @@ pipeline {
     }
     post {
         always {
+            retry(count: 3) {
```

Comment:
Can we check if we can use `terraform` retry option

Reply:
I tried an options but didn't work

Reply:
Hence going with the current solution